src/LangDic.js

```diff
@@ -1,191 +1,200 @@
-    'december':'12월',
-    'su':'일',
-    'mo':'월',
-    'tu':'화',
-    'we':'수',
-    'th':'목',
-    'fr':'금',
-    'sa':'토'
-  }
-  'es' : { // Spanish
-    'january':'Enero',
-    'february':'Febrero',
-    'march':'Marzo',
-    'april':'Abril',
-    'may':'Mayo',
-    'june':'Junio',
-    'july':'Julio',
-    'august':'Agosto',
-    'september':'Septiembre',
-    'october':'Octubre',
-    'november':'Noviembre',
-    'december':'Diciembre',
-    'su':'Do',
-    'mo':'Lu',
-    'tu':'Ma',
-    'we':'Mi',
-    'th':'Ju',
-    'fr':'Vi',
-    'sa':'Sa'
+  ko: {
+    // Korean
+    january: '1월',
+    february: '2월',
+    march: '3월',
+    april: '4월',
+    may: '5월',
+    june: '6월',
+    july: '7월',
+    august: '8월',
+    september: '9월',
+    october: '10월',
+    november: '11월',
+    december: '12월',
+    su: '일',
+    mo: '월',
+    tu: '화',
+    we: '수',
+    th: '목',
+    fr: '금',
+    sa: '토'
   },
-  'ru' : { // Russian
-    'january':'Январь',
-    'february':'Февраль',
-    'march':'Март',
-    'april':'Апрель',
-    'may':'Май',
-    'june':'Июнь',
-    'july':'Июль',
-    'august':'Август',
-    'september':'Сентябрь',
-    'october':'Октябрь',
-    'november':'Ноябрь',
-    'december':'Декабрь',
-    'su':'Вс',
-    'mo':'Пн',
-    'tu':'Вт',
-    'we':'Ср',
-    'th':'Чт',
-    'fr':'Пт',
-    'sa':'Сб'
+  es: {
+    // Spanish
+    january: 'Enero',
+    february: 'Febrero',
+    march: 'Marzo',
+    april: 'Abril',
+    may: 'Mayo',
+    june: 'Junio',
+    july: 'Julio',
+    august: 'Agosto',
+    september: 'Septiembre',
+    october: 'Octubre',
+    november: 'Noviembre',
+    december: 'Diciembre',
+    su: 'Do',
+    mo: 'Lu',
+    tu: 'Ma',
+    we: 'Mi',
+    th: 'Ju',
+    fr: 'Vi',
+    sa: 'Sa'
   },
-  'tr' : { // Turkish
-    'january':'Ocak',
-    'february':'Şubat',
-    'march':'Mart',
-    'april':'Nisan',
-    'may':'Mayıs',
-    'june':'Haziran',
-    'july':'Temmuz',
-    'august':'Ağustos',
-    'september':'Eylül',
-    'october':'Ekim',
-    'november':'Kasım',
-    'december':'Aralık',
-    'su':'Pz',
-    'mo':'Pts',
-    'tu':'Sa',
-    'we':'Ça',
-    'th':'Pe',
-    'fr':'Cu',
-    'sa':'Cts'
+  ru: {
+    // Russian
+    january: 'Январь',
+    february: 'Февраль',
+    march: 'Март',
+    april: 'Апрель',
+    may: 'Май',
+    june: 'Июнь',
+    july: 'Июль',
+    august: 'Август',
+    september: 'Сентябрь',
+    october: 'Октябрь',
+    november: 'Ноябрь',
+    december: 'Декабрь',
+    su: 'Вс',
+    mo: 'Пн',
+    tu: 'Вт',
+    we: 'Ср',
+    th: 'Чт',
+    fr: 'Пт',
+    sa: 'Сб'
   },
-}
+  tr: {
+    // Turkish
+    january: 'Ocak',
+    february: 'Şubat',
+    march: 'Mart',
+    april: 'Nisan',
+    may: 'Mayıs',
+    june: 'Haziran',
+    july: 'Temmuz',
+    august: 'Ağustos',
+    september: 'Eylül',
+    october: 'Ekim',
+    november: 'Kasım',
+    december: 'Aralık',
+    su: 'Pz',
+    mo: 'Pts',
+    tu: 'Sa',
+    we: 'Ça',
+    th: 'Pe',
+    fr: 'Cu',
+    sa: 'Cts'
+  }
+};
```

Comment:
can your changes just depend on PR's context please

Reply:
sorry, i don't understand what's wrong :(

Reply:
my bad, it wasn't understandable :)

Can you just fix Italian translation please? Maybe you can open issue for arguing on code style changes. Actually, it's a better way to integrate prettier for code formatting. I don't want to break git history more than one time because of just code formatting.

Thank you for PR ^^